### ISSUE TITLE
Merge into VoodooInput

### ISSUE THREAD

**Issue by usr-sse2** [VoodooSMBus/VoodooTrackpoint#8]
Hello! I'd like to merge this into https://github.com/acidanthera/VoodooInput, as this functionality was originally intended to be part of VoodooInput but just not implemented. I already began implementing the same, but then accidentally found this project.

**Comment by leo-labs** [comment]
Yes, I agree that merging it into VoodooInput would be a good idea. I cannot recall why we did not try to get it included. The license of this project is MIT, so it should not be a problem to relicense it under GPL 2. 

@1Revenger1 I guess that makes #7 obsolete?

**Comment by averycblack** [comment]
This is fine with me as well
Yeah I can close #7

**Comment by Sniki** [comment]
Should the merge to VoodooInput also fix this ?:
https://github.com/acidanthera/bugtracker/issues/890
